Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -4,9 +4,8 @@ const personOne = {
   favouriteFood: "Spinach",
 };
 
-// Update the parameter to this function to make it work.
-// Don't change anything else.
-function introduceYourself(___________________________) {
+
+function introduceYourself({name, age, favouriteFood} = personOne) {
```

Comment:
What does the `= personOne` do here? Why is it needed?

Reply:
```=personOne``` acts as the default array here, meaning if no other array is passed the values ```{name, age, favouriteFood}``` will come from it. It is needed here because we are only allowed to change the parameter of the function nothing else.

Reply:
What would have happened if you hadn't put the `= personOne` there?

Reply:
It would have worked fine!

Reply:
Let's remove it then! :)